models/getArticleByArticleIdModel.js

```diff
@@ -0,0 +1,24 @@
+const db = require('../db/connection.js')
+
+const getArticleByArticleId = (article_id) => {
+    const articleId = Number(article_id)
+
+    if (isNaN(articleId)) {
```

Comment:
I dont know you need to promise reject here, I think if you pass the invalid article id to the database it will reject on its own (probably with a "P2202" code :) )